drivers/include/periph/usbdev.h

```diff
@@ -355,6 +355,22 @@ typedef struct usbdev_driver {
     int (*ready)(usbdev_ep_t *ep, size_t len);
 } usbdev_driver_t;
 
+/**
+ * @brief Low level USB peripheral driver initialization
+ *
+ * This function prepares all usbdev peripherals available for initialization
+ */
+void usbdev_init_lowlevel(void);
+
+/**
+ * @brief Retrieve usbdev context from the peripheral
+ *
+ * @param num   usbdev peripheral number to retrieve
+ *
+ * @returns     the usbdev context at index @p num
+ */
+usbdev_t *usbdev_get_ctx(unsigned num);
```

Comment:
uint8_t num ?

Reply:
I'd rather keep this `unsigned`( or `unsigned int`). The size here doesn't really matter, I'd be suprised to see an mcu with more than 255 USB peripherals. With this in mind I'd like to leave it up to the platform to pick a decent size.

Reply:
+1